meta-ostro/classes/ostro-image.bbclass

```diff
@@ -10,121 +10,245 @@ IMAGE_INSTALL = " \
                 ${OSTRO_IMAGE_EXTRA_INSTALL} \
 		"
 
+# In Ostro OS, /bin/sh is always dash, even if bash is installed for
+# interactive use.
+IMAGE_INSTALL += "dash"
+
+# Certain keywords, like "iotivity" are used in different contexts:
+# - as image feature name
+# - as bundle name
+# - optional: as additional image suffix
+#
+# While this keeps the names short, it can also be a bit confusing and
+# makes some of the definitions below look redundant. They are needed,
+# though, because the naming convention could also be different.
+
+
 # Image features sometimes affect image building (for example,
 # ima enables image signing) and/or adds certain packages
 # via FEATURE_PACKAGES.
 #
-# TODO: document IoT specific image feature somewhere. Here?
-IMAGE_FEATURES[validitems] += " \
-    app-privileges \
-    autologin \
+# This is the list of image features which merely add packages.
+# This list also gets turned into the default set of swupd bundles.
+OSTRO_IMAGE_PKG_FEATURES = " \
     can \
     connectivity \
     devkit \
-    ima \
     iotivity \
     java-jdk \
     node-runtime \
     nodejs-runtime-tools \
     python-runtime \
     qatests \
-    smack \
+    ssh-server \
     soletta \
     soletta-tools \
```

Comment:
what is soletta-tools x soletta? In this image only installing the sol-fbp-runner is enough, it should bring the dependencies such as other libraries and modules. and I'd call it similar to other runtimes (python-runtime, java-runtime, soletta-runtime).

For -dev images, installing soletta-dev, which should include sol-fbp-generator and other scripts would be necessary.

Last but not least, java-jdk, node-runtime and nodejs-runtime-tools, can we have a single name pattern?